### DISCUSSION TITLE
Does the traffic involved in the pretraining for the chronos-bolt version?

### DISCUSSION THREAD

**Discussion by wyhzunzun123123** [amazon-science/chronos-forecasting#246] [Q&A] [answered]
Thank you for your excellent work which inspired me a lot！

When testing the new Chronos-bolt version， I got incredible performance on the traffic dataset with the Chronos-bolt-tiny version, making me wonder if the pertaining datasets contained it.  This is very important for me to compare the chronos with other time series foundation models fairly.  Please do not hesitate to give me any advice and information!

Best regards!

**Comment by Killer3048** [comment]
@wyhzunzun123123 

Yes, the traffic dataset was used in pretraining. You can find it here:
-> [Traffic dataset used in pretraining](https://zenodo.org/record/4656132).

For more detailed information, refer to the [Chronos paper](https://arxiv.org/abs/2403.07815), which outlines the datasets and methods used for training.
-> [Chronos paper](https://arxiv.org/abs/2403.07815)

**Comment by abdulfatir** [comment, marked as answer]
@wyhzunzun123123 @Killer3048  No, The traffic dataset was **NOT used** for pretraining. Please refer to **Table 3** (also attached below) in the appendix of the [Chronos paper](https://arxiv.org/abs/2403.07815) for details. The datasets under "Pretraining only" and "In-domain evaluation" were used for pretraining whereas the datasets under "Zero-shot evaluation" (and all other datasets not listed in the table) were not used for training. The real datasets used for training Chronos-Bolt are the same as Chronos. 

Also, @wyhzunzun123123, in future please use discussions for questions since this is not really an issue/bug. :) 

<img width="621" alt="image" src="https://github.com/user-attachments/assets/9dcb5e2f-8ec0-4a75-a528-25fd2b20f282" />

**Reply by Killer3048** [reply]
Sorry, I'm a bit scatterbrained and mixed up the tables :(. This often leads to interpretation issues. Sometimes I might even make mistakes in the code when my distraction intensifies and lasts for an hour or two, which I can’t control :)
